### ISSUE TITLE
Android/iOS: Accessibility - "Options" expand/collapse control does not announce state

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/mobile#1920]
### Steps To Reproduce

1. Turn on VoiceOver/iOS, Talkback/Android
2. Go to a screen with an "Options" expand/collapse control (e.g. "Send > Add a Send")
3. Set focus to "Options" control
4. Toggle "Options" control

### Expected Result

The control should convey its current state - whether the options are expanded or collapsed.

### Actual Result

"Options" does not convey any state information.

Further, the control is unnecessarily split into two - "Options" itself, and then the up/down chevron/arrow. The chevron button has an accessible name of just the icon, which is not announced - so the control is simply announced as "Button" - in iOS' case, VoiceOver then also announces "up arrow" or "down arrow" (as it uses extra heuristics to try and convey the unannounced button content), but not in the case of Talkback.

Ideally, "Options" should be a single tab stop/control (no extra stop for the icon), and announce as "expanded/collapsed", "open/closed", or similar.

### Screenshots or Videos

Options button when collapsed, just announced as "Options, Button"

![Options button when collapsed, just announced as "Options, Button"](https://user-images.githubusercontent.com/895831/169664055-439d2e55-39ac-474c-bde1-4bc40117fe8d.png)

Options button when expanded, just announced as "Options, Button"

![Options button when expanded, just announced as "Options, Button"](https://user-images.githubusercontent.com/895831/169664067-86a1b2d0-54d5-43fa-bad1-1cc760bf9776.png)

---

The chevron icon button - note that the accessible name (which can be seen in VO's text output at the bottom of the screen) includes an unprocessed character/icon, but it's just announced as "Button". No state announcement (expanded or collapsed) is made here either.

![The chevron icon button](https://user-images.githubusercontent.com/895831/169664077-a0eecf41-16ab-4ded-987a-e06aae1a755c.png)

VoiceOver's heuristic that then announces the chevron icon "down arrow". Talkback won't do this.

![The chevron icon button, announced as "down arrow"](https://user-images.githubusercontent.com/895831/169664115-7da7dbbc-1b1e-4cc3-b35f-8e88fa157e4f.png)

### Operating System

Android, iOS

### Build Version

2.18.0 (4572)

### Beta

- [ ] Using a pre-release version of the application.
